src/ssl.c

```diff
@@ -9084,19 +9140,23 @@ WOLFSSL_X509_EXTENSION* wolfSSL_X509_set_ext(WOLFSSL_X509* x509, int loc)
                         if (dns != NULL) {
                             if (wolfSSL_sk_GENERAL_NAME_push(sk, gn) !=
                                                           WOLFSSL_SUCCESS) {
-                            WOLFSSL_MSG("Error pushing onto stack");
-                            wolfSSL_GENERAL_NAME_free(gn);
-                            wolfSSL_sk_free(sk);
-                            sk = NULL;
```

Comment:
Not sure why this error case and the one below weren't handled with a `return NULL`, but I think they should be, hence the change.